darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -71,6 +71,40 @@ def __init__(
         References
         ----------
         .. [1] D. H. Wolpert, “Stacked generalization”, Neural Networks, vol. 5, no. 2, pp. 241–259, Jan. 1992
+
+        Examples
+        --------
+        Deterministic forecasting
```

Comment:
P3, P5

also can we use a naive and a linearregression model as forecastingmodels? to make it clear that any type of model can be used

Reply:
Should I also include covariates in the example or leave it to the ensembling notebook?

Reply:
we can leave it to the notebook